python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -121,17 +121,14 @@ def __init__(self, butler=None, refSchema=None, **kwds):
         self.makeSubtask('catalogCalculation', schema=self.measurement.schema)
 
     def runDataRef(self, dataRef, psfCache=None):
-        """!Measure a single exposure using forced detection for a reference catalog.
+        """!Organize a single exposure for forced detection for a reference catalog.
```

Comment:
I suggest reverting this change. This method does measure -- though it calls `run` to do the job. Consider appending "using a dataRef" to the old brief description, instead, to differentiate it from the brief description for `run`.